index.bs

```diff
@@ -129,9 +132,11 @@ The {{ProximitySensor/near!!attribute}} attribute of the {{ProximitySensor}}
 interface returns the result of invoking [=get value from latest reading=] with
 <emu-val>this</emu-val> and "near" as arguments.
 
+If the physical object is outside the <a>sensing range</a>, the attribute must return null.
+
 Note: If the implementation is unable to provide the [=near=] value, it could infer the [=near=]
 value from the value of [=distance=]. For example, if [=distance=] is not equal to [=max=], it
-could imply there is an object in the sensing range.
+could imply there is a physical object in the sensing range.
```

Comment:
nit: make `sensing range`  within `<a>` tag.

Reply:
Thanks! Fixed in https://github.com/w3c/proximity/pull/31/commits/658ecb504456ae63869f9e417c273d155ae6d54d